webservice/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "WebService",
+    "summary": """
+        Defines webservice abstract definition to be used generally""",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
+    "development_status": "Alpha",
```

Comment:
Can you change this to Beta at least?

@etobella maybe more?

Reply:
I think we can put as Mature and add me as maintainer WDYT?

Reply:
I agree, it is done!